chatApp.js

```diff
@@ -19,12 +19,15 @@ io.on("connection", (socket) => {
   socket.on(NEW_CHAT_MESSAGE_EVENT, async (data) => {
     try {
       await axios.post(`${BACKEND_SERVER}party/${data.partyId}/chat`, data)
+      var currentDate = new Date();
```

Comment:
didn't you say this was autogen from Parse? Might be worth looking this up on stackoverflow to see how people usually handle this